teuthology/orchestra/daemon/systemd.py

```diff
@@ -166,11 +166,7 @@ def running(self):
         Are we running?
         :return: The PID if remote run command value is set, False otherwise.
         """
-        pid = self.pid
-        if pid > 0:
-            return pid
-        else:
-            return None
+        return self.pid
```

Comment:
this change not only address the compare-to-none issue, it also changes the semantics of this function? could you help me understand why it is safe to do so?

Reply:
Since the `self.pid` returns `None` or `integer` value, it doesn't much affect the return value in `running` method.

Reply:
i mean, you have an assumption that `pid` is either "a number greater than 0" or None in this change, which does not hold before your change.

Reply:
Thanks @tchaikov . I have updated the code.